src/SonarScanner.MSBuild.TFS/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="COVXML_DIAG_SkippingCoverageCheckPropertyProvided" xml:space="preserve">
+    <value>Property 'sonar.cs.vscoveragexml.reportsPaths' provided, skipping the search of coveragexml file in default folders...</value>
```

Comment:
Minor: "skipping the search _for_"

Same for the TRX string.

Reply:
That has been corrected.